### ISSUE TITLE
Add a rule to disallow extra newlines before a block closes

### ISSUE THREAD

**Issue by tremby** [sasstools/sass-lint#1073]
The following currently produces no warnings or errors:

```scss
.container {
  button {
    display: block;

  }

}
```

I expect warnings for lines 4 and 6, since I don't want those extra blank lines, even though blank lines between blocks should be allowed.

Am I missing an existing rule which will do this? Or perhaps one could be added.

**Comment by saltire** [comment]
Here's the ESLint equivalent: https://eslint.org/docs/rules/padded-blocks
